### ISSUE TITLE
bro what happened to html ports? If you can bring them back please 

### ISSUE THREAD

**Issue by LEVIXDDLMAO** [ZHyproX/Indie-Cross-Master#1]

**Comment by ZHyproX** [comment]
ı was busy

**Comment by LEVIXDDLMAO** [comment]
Nah your fine i was just wondering i was able to get the extra keys done with the google drive so i wanna to do the rest and make website with you as the main html ported

**Comment by LEVIXDDLMAO** [comment]
![image](https://user-images.githubusercontent.com/92174516/163393632-074afd29-0fb1-41d4-a268-87ef48108da6.png)
 this is the error i get when i try to download them

**Comment by LEVIXDDLMAO** [comment]
and can you keep them on google drive

**Comment by LEVIXDDLMAO** [comment]
can you fix them?

**Comment by ZHyproX** [comment]
actually psych 0.5.2 is out just its didnt let me build so download downloadable version

**Comment by LEVIXDDLMAO** [comment]
yeah psych 0.5.2h is too bliud, and im talking about the acc hank and changelle edd and Uberkids and psychic

**Comment by ZHyproX** [comment]
they sucks

**Comment by ZHyproX** [comment]
ı mean the port

**Comment by ZHyproX** [comment]
ı need more stroage so

**Comment by LEVIXDDLMAO** [comment]
Ah i see

**Comment by LEVIXDDLMAO** [comment]
but is it possable to send me the flies or something? I tried adding you on discord but heres my email if you can send them Dontblock012@outlook.com
